src/__tests__/activities.test.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * @jest-environment jsdom
+ */
+/* eslint-disable*/
+import ActivitiesManager from '../__mocks__/activities.js';
+
+describe('Task1', () => {
+
+  it('Testing activities Manager Add', () => {
+    document.body.innerHTML = 
+    '<div>' +
```

Comment:
- [ ] in this case, you are only testing the mock functions you wrote instead of the actual code.

Reply:
Ok yeah that is true so if we just import from the normal file that should be okay right? to test not the mocks but the real one @mikethreels